components/infobox/commons/infobox_league.lua

```diff
@@ -0,0 +1,245 @@
+								args.organizer4, args['organizer-name4'], args['organizer-link4'], args.organizerref4)
+                        )
+                        :make()
+            )
+            :cell('Sponsor(s)', args.sponsor)
+            :fcell(Cell  :new('Mode')
+                        :options({})
+                        :content(args.mode)
+                        :variables({key = 'mode', value = args.mode})
+                        :make()
+            )
+            :cell('Server', args.server)
+            :cell('Misc. mode', args.miscmode)
+            :fcell(Cell  :new('Type')
+                        :options({})
+                        :content(args.type)
+                        :categories(
+                            function(_, ...)
+                                local value = select(1, ...)
+                                value = tostring(value):lower()
+                                if value == 'offline' then
+                                    infobox:categories('Offline Tournaments')
+                                elseif value == 'online' then
+                                    infobox:categories('Online Tournaments')
+                                end
+                            end
+                        )
+                        :variables({key = 'tournament_type', value = args.type})
+                        :make()
+            )
+            :cell('Venue', args.venue)
+            :fcell(self:createPrizepool(args):make())
+            :fcell(Cell :new('Date')
+                        :options({})
+                        :content(args.date)
+                        :variables(
+                            {key = 'date', value = self:_cleanDate(args.date)},
+                            {key = 'tournament_date',
+								value = self:_cleanDate(args.edate) or self:_cleanDate(args.date) or ''}
+                        )
+						:make()
+            )
+            :fcell(Cell :new('Start Date')
+                        :options({})
+                        :content(args.sdate)
+                        :variables(
+                            {key = 'sdate', value = self:_cleanDate(args.sdate)}
+                        )
+						:make()
+            )
+            :fcell(Cell :new('End Date')
+                        :options({})
+                        :content(args.edate)
+                        :variables(
+                            {key = 'edate', value = self:_cleanDate(args.edate)}
+                        )
+						:make()
+            )
+            :fcell(self:createTier(args):make())
+    League:addCustomCells(infobox, args)
+
+    local links = Links.transform(args)
+
+    infobox :header('Links', not Table.isEmpty(links))
+            :links(links)
+            :centeredCell(args.footnotes)
+            :chronology({
+                previous = args.previous,
+                next = args.next,
+                previous2 = args.previous2,
+                next2 = args.next2,
+            })
```

Comment:
could we add previous3 and next3?

and could we make that a function?
on sc2 we have it semi-automated for previous and next (the unnumbered ones), maybe we could even add it to the default:

```
local next, previous = p.computeChronology(args)

-- Automatically fill in next/previous for touranaments that are part of a series
function p.computeChronology(args)
	-- Criteria for automatic chronology are 
	-- - part of a series and numbered
	-- - the subpage name matches the number
	-- - prev or next are unspecified
	-- - and not suppressed via auto_chronology=false
	local title = mw.title.getCurrentTitle()
	local number = tonumber(title.subpageText or '')
	local automateChronology = 
		(args.series or '') ~= ''
		and number ~= nil
		and tonumber(args.number or '') == number
		and title.subpageText ~= title.text
		and args.auto_chronology ~= 'false'
		and ((args.next or '') == '' or (args.previous or '') == '')

	if automateChronology then
		local previous = (args.previous or '') ~= '' and args.previous
		local next = (args.next or '') ~= '' and args.next
		local nextPage = (args.next or '') == '' and title.basePageTitle:subPageTitle(tostring(number + 1)).fullText
		local previousPage = (args.previous or '') == '' and title.basePageTitle:subPageTitle(tostring(number - 1)).fullText

		if not next and p._exists(nextPage) then
			next = nextPage .. '|#' .. tostring(number + 1)
		end

		if not previous and 1 < number and p._exists(previousPage) then
			previous = previousPage .. '|#' .. tostring(number - 1)
		end

		return next, previous
	else
		return args.next, args.previous
	end
end

```

Reply:
Let's leave this for a follow-up PR